utils/importWikipedia.php

```diff
@@ -389,15 +389,16 @@ function nominatimXMLEnd($hParser, $sName)
             break;
         case 'prefecture japan':
             $aRecord['name'] = trim(str_replace(' Prefecture', ' ', $aRecord['name']));
+            break;
```

Comment:
This looks like the fallthrough is intended. Like all case statements directly below a prefecture is a state.